### PR TITLE
maths magician: event

### PR DESCRIPTION
**Pull request by Moses-chibuike**
- Created logic directory and in-turn created calculate.js and operate.js files inside it
- Copied the contents of calculate.js and operate.js into the logic directory
- Install big.js using NPM
- Implemented the event handlers needed to use the math logic from calculate.js and operate.js in the React component.
- Test the app in browser to make sure that the calculator performs basic operations

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Moses, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure to remove the Live demo link from the ReadMe if the app is not yet deployed.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,49 +1,59 @@
 /* eslint-disable react/button-has-type */
```
Comment (Lucas-Masaba) [inline src/components/Calculator.js]: - [ ] Please remove the eslint disable comment here. It disables linter checks which degrade the quality of your code.

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @Moses-chibuike  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing :check_
- Descriptive pull request :scroll: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Writing better and more commit messages

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (peterrobert) [reply src/components/Calculator.js]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @Moses-chibuike , 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Moses-chibuike** [comment]
> # Hello @Moses-chibuike 👋 ,
> Good job so far! 👍 There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> almost-there [ almost-there ](https://camo.githubusercontent.com/4187074f6b8307d11768c326133ea198a73743e5b747571572effcf028fa3f59/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f52657944344f64526948654c4443526145502f67697068792e676966) [ ](https://camo.githubusercontent.com/4187074f6b8307d11768c326133ea198a73743e5b747571572effcf028fa3f59/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f52657944344f64526948654c4443526145502f67697068792e676966)
> 
> ## Highlights 🍾
> * Linter checks passing :check_
> * Descriptive pull request 📜
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions 💭
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> You can also consider:
> 
> * Writing better and more commit messages
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

okay thanks

**Comment by Moses-chibuike** [comment]
> > # Hello @Moses-chibuike 👋 ,
> > Good job so far! 👍 There are some issues that you still need to work on to go to the next project but you are almost there!
> > almost-there [ almost-there ](https://camo.githubusercontent.com/4187074f6b8307d11768c326133ea198a73743e5b747571572effcf028fa3f59/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f52657944344f64526948654c4443526145502f67697068792e676966) [ ](https://camo.githubusercontent.com/4187074f6b8307d11768c326133ea198a73743e5b747571572effcf028fa3f59/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f52657944344f64526948654c4443526145502f67697068792e676966)
> > ## Highlights 🍾
> > 
> > * Linter checks passing :check_
> > * Descriptive pull request 📜
> > 
> > ## Required Changes ♻️
> > _Check the comments under the review._
> > ## Optional suggestions 💭
> > _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> > You can also consider:
> > 
> > * Writing better and more commit messages
> > 
> > Cheers and Happy coding!👏👏👏
> > Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.
> > **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> > _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
>

**Pull request merged**